x/leverage/keeper/token.go

```diff
@@ -71,7 +73,7 @@ func (k Keeper) GetTokenSettings(ctx sdk.Context, denom string) (types.Token, er
 
 // SaveOrUpdateTokenSettingsToRegistry adds new tokens or updates the new tokens settings to registry.
```

Comment:
```suggestion
// SaveOrUpdateTokenSettingsToRegistry adds new tokens or updates the new tokens settings to registry.
// It requires maps of the currently registered base and symbol denoms, so it can prevent duplicates of either.
```